src/dmd/expressionsem.d

```diff
@@ -6272,7 +6272,7 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
             /* Rewrite to get rid of the comma from rvalue
              */
             if (!(cast(CommaExp)exp.e2).isGenerated)
-                exp.deprecation("Using the result of a comma expression is deprecated");
+                exp.error("The result of a comma expression cant be used");
```

Comment:
ditto

Reply:
Well cant is the existing DMD style:

```
> ag "cant" | wc -l
466 
> ag "can't" | wc -l
128
```

Reply:
Existing bad grammar should not be justification for adding more bad grammar!

Reply:
I have seen @WalterBright repeatedly use "cant" - even in recent PRs. That was my motivation to use "cant".

Reply:
Examples:
- https://github.com/dlang/dmd/pull/6092
- https://github.com/dlang/dmd/pull/7307

Reply:
"cant" is not a word.  I believe @WalterBright made a mistake in those PRs.

Reply:
Yah let's fix those. Thanks!

Reply:
Fair enough, I changed it to can't.

Reply:
@JinShil Actually, "cant" *is* a word... but it has a [different meaning](https://en.wiktionary.org/wiki/cant). :-D 

@wilzbach The next time Walter submits a PR with the wrong use of "cant", somebody should point it out so that it can be fixed. :-P